### DISCUSSION TITLE
Unhandled Exception are not getting printed when using kubernetes Executor

### DISCUSSION THREAD

**Discussion by bazingarj** [apache/airflow#33995] [General]
Airflow version: 2.4.3

**Issue:** When i use local executor i am able to see the exception in airflow logs And when i use kubernetes Executor airflow spawns a Pod for each Dag task but the dag gets failed due to some error which is not getting printed in the logs.

**Tried So far**
* Setting up Debug Mode
* try to catch error on on_failure_callback

Please help getting stuck with failed pods with no error shown.

**Comment by potiuk** [comment]
Did you try task logs in the UI ? What do they show?

**Reply by bazingarj** [reply]
it is not showing there as well

**Comment by potiuk** [comment]
There were some mprovement to how logs are pulled since 2.4 so I suggest to upgrade to latest version it's hard to help you when there are no logs of any sort.

**Reply by bazingarj** [reply]
Sure i would deploy on latest version
